sstable/src/block_match_automaton.rs

```diff
@@ -132,7 +132,7 @@ fn match_range_start<S, A: Automaton<State = S>>(
     automaton: &A,
     mut state: S,
 ) -> bool {
-    // case [abcdgj, abcpqr], `abcd` is already consumed, we need to handle:
+    // case ]abcdgj, abcpqr], `abcd` is already consumed, we need to handle:
```

Comment:
I prefer your notation, unfortunately in my experience, non-french people seem more familiar with `(left, right]`.